torchci/components/metrics/panels/TimeSeriesPanel.tsx

```diff
@@ -289,13 +356,65 @@ export default function TimeSeriesPanel({
     granularity,
     groupByFieldName,
     timeFieldName,
-    yAxisFieldName
+    yAxisFieldName,
+    false,
```

Comment:
Any reason why this is false?  I think it might be messing up the metrics page

Reply:
mistake, wanted to keep it the default, changed now